src/main/java/org/bankofspring/validator/BankValidator.java

```diff
@@ -0,0 +1,30 @@
+package org.bankofspring.validator;
+
+import org.bankofspring.entities.Account;
+
+public class BankValidator {
+
+	/**
+	 * This should probably return the errors at some point.
+	 * @param fromAccount
+	 * @param toAccount
+	 * @param amount
+	 * @return
+	 */
+	public boolean validateTransfer(Account fromAccount, Account toAccount, long amount) {
+		// ensure the from account has money
+		if (fromAccount.getAccountAmount() < amount) {
+			return false;
+		}
+		
```

Comment:
Good point, however can you imagine the response that you'd get if you rejected a payment into someone's account because they had too much money. :smile: 

You'd need to have £92,233,720,368,547,758.07 (I guess it depends on the currency. I'm sure this is more than all the money in the world though)